service/src/main/java/org/lfenergy/compas/scl/auto/alignment/builder/VoltageLevelGraphBuilder.java

```diff
@@ -94,36 +96,39 @@ private Optional<GenericPowerTransformer> getPowerTransformer(String pathName) {
         return Optional.empty();
     }
 
-    private void processConductingEquipment(GenericConductingEquipment conductingEquipment) {
+    private void processConductingEquipment(GenericConductingEquipment conductingEquipment, int order) {
         var terminals = conductingEquipment.getTerminals();
         var fullName = conductingEquipment.getFullName();
-        var node = createSwitchNode(fullName);
+        var node = createSwitchNode(fullName, order);
 
         if (!terminals.isEmpty()) {
-            Node node1 = terminalToNode(terminals.get(0));
+            Node node1 = terminalToNode(terminals.get(0), order);
             Node node2 = null;
             var termNb = terminals.size();
             if (termNb == 1) {
-                node2 = createLoad(fullName + "/Grounded");
+                node2 = createLoad(fullName + "/Grounded", order);
             } else if (termNb == 2) {
```

Comment:
node2 can be null, (if termNb does not equals 1 or 2). should this be handled?

Reply:
I don't think it's possible, because you 0, 1 or 2 terminals. 0 is already checked a few lines before.
But I changed the code to make more error-prone.